### PR TITLE
Use @npm instead of @ngdeps now that downstream angular build uses angular bundles

### PR DESCRIPTION
**Pull request by gregmagolan**
Assuming this will conflict with 7.x branch so set target to master-only. Once it lands I'll make a patch PR.

### CODE REVIEW

File: BUILD.bazel [hunk 1]
```diff
@@ -33,34 +33,18 @@ filegroup(
...
+        "@npm//node_modules/angular-1.5:angular.min.js",
+        "@npm//node_modules/angular-1.6:angular.js",
+        "@npm//node_modules/angular-1.6:angular.min.js",
+        "@npm//node_modules/angular-mocks:angular-mocks.js",
+        "@npm//node_modules/angular-mocks-1.5:angular-mocks.js",
+        "@npm//node_modules/angular-mocks-1.6:angular-mocks.js",
     ],
 )
 
-# A nodejs_binary for @angular/bazel/ngc-wrapped to use by default in
```
Comment (gregmagolan) [inline BUILD.bazel]: one less hack :)

File: WORKSPACE [hunk 2]
```diff
@@ -15,22 +15,13 @@ load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
...
-# Use a mock @npm repository while we are building angular from source
-# downstream. Angular will get its npm dependencies with in @ngdeps which
-# is setup in ng_setup_workspace().
-# TODO(gregmagolan): remove @ngdeps once angular is no longer build from source
-#   downstream and have build use @npm for npm dependencies
-local_repository(
-    name = "npm",
-    path = "tools/npm_workspace",
+    sha256 = "5c86b055c57e15bf32d9009a15bcd6d8e190c41b1ff2fb18037b75e0012e4e7c",
+    urls = ["https://github.com/bazelbuild/rules_nodejs/releases/download/0.26.0/rules_nodejs-0.26.0.tar.gz"],
```
Comment (gregmagolan) [inline WORKSPACE]: update to 0.26.0 of rules needed to pick up `data` files fix for `yarn_install`

**Comment by gregmagolan** [comment]
Also updates to bazel rules 0.26.0 as a data files fix for yarn_install is needed

**Review by alexeagle — APPROVED** [review]

**Review by IgorMinar — APPROVED** [review]
global approval

**Comment by alexeagle** [comment]
Caretaker assistance: this has the Global Approve of Igor

**Pull request closed without merging**